### ISSUE TITLE
common/documentserver/home/server/FileConverter/bin/x2t missing.

### ISSUE THREAD

**Issue by qianwch** [ONLYOFFICE/build_tools#885]
It seems that some target binaries were not built. So the document-server-package (make rpm) could not succeed.

```
chmod ug=r common/documentserver/config/*.json
cp -fr -t common/documentserver/home common/documentserver/home/server/LICENSE.txt  common/documentserver/home/server/3rd-Party.txt  common/documentserver/home/server/license
rm -fr common/documentserver/home/server/LICENSE.txt  common/documentserver/home/server/3rd-Party.txt  common/documentserver/home/server/license
[ -f ./common/documentserver/license/onlyoffice-documentserver/LICENSE.txt ] \
        && cp -f -t common/documentserver/home ./common/documentserver/license/onlyoffice-documentserver/LICENSE.txt \
        || cp -f -t common/documentserver/home LICENSE.txt
chmod u+x common/documentserver/home/server/FileConverter/bin/x2t
chmod: cannot access 'common/documentserver/home/server/FileConverter/bin/x2t': No such file or directory
Makefile:322: recipe for target 'documentserver' failed
make: *** [documentserver] Error 1
```

**Comment by igwyd** [comment]
Hello @qianwch, as i can see x2t is in folder ```...\out\linux_64\onlyoffice\documentserver\server\FileConverter\bin\x2t``` after compilation. Describe in more detail how do you got this issue?

**Comment by qianwch** [comment]
> Hello @qianwch, as i can see x2t is in folder `...\out\linux_64\onlyoffice\documentserver\server\FileConverter\bin\x2t` after compilation. Describe in more detail how do you got this issue?

I ran the build like this:  
```
./automate.py --branch v8.2.2.26
```

**Comment by igwyd** [comment]
There may be issues with build 26, we released 22, try to build it v8.2.2.22.

**Comment by qianwch** [comment]
> There may be issues with build 26, we released 22, try to build it v8.2.2.22.

It worked!
